package.json

```diff
@@ -77,7 +77,7 @@
   },
   "homepage": "https://github.com/join-monster/join-monster#readme",
   "peerDependencies": {
-    "graphql": "^15.4.0"
+    "graphql": "^16.6.0"
```

Comment:
If users are on 3.1.1 with graphql@15 and they do `npm i`, they'll get a peer dep error, right? I don't want to break their install unless they upgrade to a v4. I'm thinking either:

1. peerDep of v15 || v16, or
2. just v16 and releasing as `4.0.0-alpha`

1 seems preferable?

Reply:
yes. thanks for the feedback.  #1 seems preferable. let me test that and pursue any appropriate changes.

Reply:
hi 
So I'm finally getting around to working on the first comment above. I changed the json.package file to indicate a dependency on v15 or v16 as
```
"peerDependencies": {
    ""peerDependencies": {
    "graphql": "^16.6.0|^15.4.0"
  },
graphql": "^16.6.0|^15.4.0"
  },
```
and want to ensure I'm testing it properly.
I just manually ran the main set of test suites with `graphql@v16.6.0` and then separately with  `graphql@15.4.0` installed. This was useful because it forced me to revert `graphql-relay` from` 0.10.0` back down to the original `0.6.0`, else something else broke when at `graphql@15.4.0` .
Let me know if this doesn't cover the intent of your comment.